tesseract/tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h

```diff
@@ -699,7 +710,7 @@ struct DiscreteBroadphaseContactResultCallback : public BroadphaseContactResultC
                            int /*index1*/) override
   {
     if (cp.m_distance1 > static_cast<btScalar>(contact_distance_))
-      return 0;
+      return 0;  // LCOV_EXCL_LINE
```

Comment:
There are a bunch of lines like this. Why are these being excluded?

Reply:
I do not believe this should ever be reached but left it here. I am not sure if we should add an assert.

Reply:
I removed these ones.

Reply:
Well if they are useful to keep that's fine. Don't feel pressured to remove it if you think it is useful. I just wanted to make sure we weren't checking a condition that will never happen or excluding from coverage something that could sometimes happen.

Reply:
I will remove it until I have time to further investigate and confirm that this never happens.